application/classes/Ushahidi/Formatter/Post/GeoJSONCollection.php

```diff
@@ -47,6 +47,20 @@ public function __invoke($entities)
 
 			if (! empty($geometries))
 			{
+				$set_name = [];
+				$set_ids = $entity->sets;
+
+				if ($set_ids != null) {
+					$query = DB::select('sets.name')
```

Comment:
I would revise part of the geojson output as below (which was `set_name`).
`"collections": [
                    {
                      "id": 1,
                      "name": "Taipei Fire"
                    },
                    {
                      "id": 2,
                      "name": "Taipei Flood"
                    }
                ]`
Is this format fit the requirement? Thanks